metric/metric.go

```diff
@@ -22,6 +26,21 @@ type InstanceCountMetric struct {
 
 // IssueCountMetric is the API message for bb.issue.count
 type IssueCountMetric struct {
-	Type  api.IssueType
-	Count int
+	Type   api.IssueType
+	Status api.IssueStatus
+	Count  int
+}
+
+// ProjectCountMetric is the API message for project count metric
+type ProjectCountMetric struct {
+	TenantMode   api.ProjectTenantMode
+	WorkflowType api.ProjectWorkflowType
+	Count        int
+}
+
+// PolicyCountMetric is the API message for policy count metric
+type PolicyCountMetric struct {
+	Type          api.PolicyType
+	EnvironmentID int
```

Comment:
EnvironmentID as an integer type might be less-informative. We may want to use the name? Even though, the named strings are not consistent across different customers.

Reply:
We will get the environment by id here https://github.com/bytebase/bytebase/pull/1344/files#diff-d06b4ebe25f136cee0c097e1e0d191209ffd261918f7305dc9cd1f76004fc5d8R38

But yes it's a question that different customers may use different env names. We may need to discuss this in our group chat.

Reply:
ID is not useful, we need to collect the environment name instead.

Reply:
We did collect the env name here https://github.com/bytebase/bytebase/pull/1344/files#diff-d06b4ebe25f136cee0c097e1e0d191209ffd261918f7305dc9cd1f76004fc5d8R38

Or it's also acceptable to move the code into the store

Reply:
Update. As we need to count policy by the type and payload value, we cannot use SQL to count data directly, need to fetch the policy list and then loop and count. But as one environment can only have one policy, seems it's acceptable to list all policies.